src/sass/patterns/_form-elements.scss

```diff
@@ -70,3 +70,17 @@
     }
   }
 }
+
+[data-form='search-issues'] {
+  gap: $spacing-x-small;
+  grid-template-rows: min-content auto;
+
+  .slds-form-element {
+    grid-column: 1;
+    grid-row: 1 / span 2;
+  }
+
+  .issues-search-button {
+    grid-column: 2;
+  }
```

Comment:
@dvdherron I don't think we need to specify this if the markup doesn't change?

Reply:
removed